### PR TITLE
Added changelog #196

### PR DESCRIPTION
**Pull request by ivarconr**
Something more to add @sveisvei @vsandvold ?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+## 2.0.0 (Januar 2017)
```
Comment (SimenB) [inline CHANGELOG.md]: January

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Support multiple strategies. This makes it easy to use multiple activation strategies in combination.
+- Client metrics. Gives details about what toggles a specific client application uses, how many times a toggle was evaluated to true / false. Everything presented in the UI. 
+- Client registration. This gives insight about connected clients, instances, strategies they support. 
+- Client Application overview. Based on metrics and client registrations.
+- Publish unleash-server to npm. 
+- Provide Prometheus endpoint for service metrics (response times, memory usage, etc).
+- A lot of bug-fixes (check commit history and issues for reference)
+- Unleash-frontend as a separate repo: https://github.com/Unleash/unleash-frontend. Total rewrite of UI using react + redux + material desing.
```
Comment (SimenB) [inline CHANGELOG.md]: Design

**Comment by vsandvold** [comment]
Awesome job, folks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- Unleash moved to it’s own organization: https://github.com/Unleash making it more open and allow everyone to contribute. 
+- Unleash-docker as a separate module: https://github.com/Unleash/unleash-docker 
+- Unleash binary, making it easy to install and use Unleash as a service. 
+- Removed all config/tuning that was specific to FINN.no usage of Unleash.
+
+**If you are migrating from 1.0.0 to 2.0.0 we recommend reading [the migration guide](https://github.com/Unleash/unleash/blob/master/docs/migration-guide.md)**
+
+
+
+## 1.0.0 (Januar 2014)
```
Comment (SimenB) [inline CHANGELOG.md]: January here as well :D

**Comment by SimenB** [comment]
LGTM

**Comment by sveisvei** [comment]
gg

**Pull request merged**